help/md/dnCauchy.md

```diff
@@ -3,15 +3,14 @@ dnCauchy
 ## title
 Cauchy Distribution
 ## description
-Cauchy distribution with location equal to ‘location’ and scale equal to ‘scale’.
+The Cauchy probability distribution describes the distribution of the ratio of two independent normal variables with mean 0 and variance 1. 
 ## details
-The Cauchy distribution has density:
-
-f(x) = 1/(pi * scale) * 1/(1 + x-(location/scale)^2)
-
+The Cauchy distribution takes on two parameters, location and scale. The probability density function is f(x) = 1/(pi * scale) * 1 / (1 + x-(location/scale)^2)
```

Comment:
We could also say that it is symmetric, but has broad enough tails that the mean and variance do not exist.